couch/check.py

```diff
@@ -203,27 +278,39 @@ def _get_instance_names(self, server, instance):
         else:
             return [name]
 
+    def _get_dbs_to_scan(self, server, instance, name, tags):
```

Comment:
Can you add a few comments here detailing what the function is doing and why? While I think I understand the function and how it distributes the work, I am not positive that I do, and I don't want future contributors to break this by accident.